feathr_project/test/test_feature_registry.py

```diff
@@ -89,7 +91,25 @@ def entity_array_to_dict(arr):
     inputs = registry.search_input_anchor_features(['hierarchical_derived_feature'],entity_array_to_dict(anchors+[derived_feature_with_multiple_inputs,hierarchical_derived_feature]))
     assert len(inputs)==3
     assert "input_anchorA" in inputs and "input_anchorB" in inputs and "input_anchorC" in inputs
-    
+
+def test_feathr_get_sample_features_from_registry():
+    runner = CliRunner()
```

Comment:
Two comments: 
1. We should put this check in the same test with `test_feathr_register_features_e2e`, otherwise it's hard to ensure that the features are already registered.
2. can we compare the returned object with the original object? Either thru __eq__ (https://docs.python.org/3.8/reference/datamodel.html#object.__eq__) or do that manually, to make sure the values of the objects are the same.

Reply:
Fixed